Source: Niels Ole Salscheider
Upstream: upstream bug report: https://bugs.meego.com/show_bug.cgi?id=25475
Reason: without this, tests fail

--- a/po/POTFILES.in	2012-03-30 11:45:48.000000000 +0200
+++ b/po/POTFILES.in	2012-07-10 23:25:54.405539860 +0200
@@ -15,3 +15,4 @@
 src/gnome-bluetooth/syncevolution.c
 src/dbus/server/notification-backend-libnotify.cpp
 src/dbus/server/auto-sync-manager.cpp
+src/synthesis/src/pcre/pcre_compile.c
